src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -301,7 +302,7 @@ void SwerveChassis::UpdateOdometry()
             wpi::array<double, 3> visionMeasurementStdDevs = visionPose.value().visionMeasurementStdDevs;
             units::length::meter_t poseDifference = chassisPose2d.Translation().Distance(visionPose2d.Translation());
 
-            if ((visionMeasurementStdDevs[0] == 0.5) || (poseDifference < units::length::meter_t(0.5) && visionMeasurementStdDevs[0] == 1.0) || (poseDifference < units::length::meter_t(0.3) && visionMeasurementStdDevs[0] == 2.0))
+            if ((visionMeasurementStdDevs[0] == 0.5) || (poseDifference < units::length::meter_t(0.5) && visionMeasurementStdDevs[0] == 1.0) || (poseDifference < units::length::meter_t(0.3) && visionMeasurementStdDevs[0] == 2.0) && !frc::DriverStation::IsTeleopEnabled())
```

Comment:
good for now, but we probably want to update when launching as the vision data is probably good

Reply:
I'm going to clean up warnings on another branch